cephprimarystorage/cephprimarystorage/cephagent.py

```
@@ -696,13 +696,15 @@ def retry(times=3, sleep_time=3):
             def wrap(f):
                 @functools.wraps(f)
                 def inner(*args, **kwargs):
+                    ex = None
                     for i in range(0, times):
```

Ruff (0.15.4)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)